Library/Formula/yelp-tools.rb

```diff
@@ -0,0 +1,20 @@
+class YelpTools < Formula
+  homepage "https://github.com/GNOME/yelp-tools"
+  desc "Tools that help create and edit Mallard or DocBook documentation."
+  url "https://download.gnome.org/sources/yelp-tools/3.16/yelp-tools-3.16.1.tar.xz"
+  sha256 "b4f66c145af1c6448dc51037d305d6844da13dc31d07729b8e29005ee4fef89c"
+
+  depends_on "itstool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "yelp-xsl" => :build
```

Comment:
I think the yelp-xsl dependency is a runtime one too after looking at macports and the RPM spec files. Will fix this.

I think it's best to keep both packages separate since they are officially two separate projects and everybody else seems to honor this... If someone would ever add yelp (the gnome help viewer), that would have a dependency on yelp-xsl, but not on yelp-tools.